src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionDetailsTest.java

```diff
@@ -0,0 +1,64 @@
+package teammates.test.cases.datatransfer;
+
+import java.util.HashMap;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.datatransfer.questions.FeedbackMcqQuestionDetails;
+import teammates.common.datatransfer.questions.FeedbackQuestionDetails;
+import teammates.common.datatransfer.questions.FeedbackQuestionType;
+import teammates.common.datatransfer.questions.FeedbackResponseDetails;
+import teammates.test.cases.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackQuestionDetails}
+ *      {@link FeedbackMcqQuestionDetails}.
+ */
+public class FeedbackQuestionDetailsTest extends BaseTestCase {
+
+    @Test
+    public void testCreateQuestionDetailsTest() {
```

Comment:
Use our naming convention?

Reply:
I tried to follow the testing style in `FeedbackResponseDetailsTest.java` and hence the naming.

> You are testing too many things in a single test.

I am planning to make a new test for each question type and try to test all untested methods for that question type in the particular test. Does this sound okay? Will name the test functions accordingly following the naming conventions